data.py

```diff
@@ -0,0 +1,22 @@
+import mysql.connector
+
+# Setting up the Database
+mydb = mysql.connector.connect(
```

Comment:
You should always use environment variables for DB secrets

Reply:
Updated ! Thanks